core/vm/interpreter.go

```diff
@@ -39,6 +39,13 @@ type Config struct {
 	// may be left uninitialised and will be set to the default
 	// table.
 	JumpTable [256]operation
+
+	// Enable support for EWASM contracts
+	EWASMEnabled bool
+	// Type of the EWASM interpreter ("wagon" or "evmc")
+	EWASMInterpreter string
+	// Type of the EVM interpreter ("geth" or "evmc")
```

Comment:
Please get rid of `geth` and `wagon`. Empty means built in, non empty means use something else.